Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM openjdk:11-jre
```

Comment:
Use of openjdk docker images has been [deprecated](https://hub.docker.com/_/openjdk) for a while and is not recommended. This should be replaced by Temurin/Corretto or other alternatives.

Reply:
Migrated from `openjdk:11-jre` to `amazoncorretto:11-al2023`